src/cascadia/TerminalControl/IMouseWheelListener.idl

```diff
@@ -11,6 +11,6 @@ namespace Microsoft.Terminal.Control
     [uuid("65b8b8c5-988f-43ff-aba9-e89368da1598")]
     interface IMouseWheelListener
     {
-        Boolean OnMouseWheel(Windows.Foundation.Point coord, Int32 delta, Boolean leftButtonDown, Boolean midButtonDown, Boolean rightButtonDown);
+        Boolean OnMouseWheel(Windows.Foundation.Point coord, Boolean horizontal, Int32 delta, Boolean leftButtonDown, Boolean midButtonDown, Boolean rightButtonDown);

```

Comment:
Instead of passing a bool everywhere, what if we changed `Int32 delta` to `Windows.Foundation.Point delta` instead?

Reply:
Will do